keras_preprocessing/image/dataframe_iterator.py

```diff
@@ -161,6 +161,10 @@ def __init__(self,
                                                 batch_size,
                                                 shuffle,
                                                 seed)
+        root = self.directory or ''
+        self.path_to_files = [
```

Comment:
Can you call the attribute `self.filepaths`? It is a common practice to add a prefix underscore if the attribute will be exposed via a property, thanks. 
Perhaps we do not need to create that intermediate `root` variable.
and can you change this logic above the instantiation of the parent class? Line 159

Reply:
Yes, I fixed my code according to your remark